plugins/ecs-bridge/engine/engine.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
We could actually do better and break this up into:
```
type Bridge interface {
Create()
CreateVeth()
AttachHostVethInterface()
Configure()
}

type RunIPAM interface {
Add()
Del()
}

type Veth interface {
ConfigureContainer()
GetInterfaceIPV4Address()
Delete()
}
```

But, I don't want to do it right now and can create an issue to follow it up if you like this refactor.